filament/backend/src/opengl/OpenGLContext.cpp

```diff
@@ -138,6 +138,23 @@ OpenGLContext::OpenGLContext() noexcept {
     glHint(GL_FRAGMENT_SHADER_DERIVATIVE_HINT, GL_NICEST);
 #endif
 
+#if !defined(NDEBUG) && defined(GL_KHR_debug)
+    if (ext.KHR_debug) {
+        auto cb = [](GLenum source, GLenum type, GLuint id, GLenum severity, GLsizei length,
+                const GLchar* message, const void *userParam) {
+            slog.e << "KHR_debug: " << message << io::endl;
```

Comment:
don't we want to log the source/type/id/severity ?
and/or should we do a slog.{i|d|e} based on severity?